sunpycube/cube/NDCube.py

```diff
@@ -10,8 +12,9 @@
 import copy
 
 DimensionPair = namedtuple('DimensionPair', 'lengths axis_types')
```

Comment:
I'm sorry @abit2.  Can I be a pain and ask you to change ```'lengths'``` in ```DimensionPair``` to ```shape```?  On reflection, I think ```lengths``` is not very good.  My bad.